.github/workflows/publish_release.yaml

```
@@ -0,0 +1,20 @@
+name: Publish release
+on:
+    push:
+        tags:
+            - "v[0-9]+.[0-9]+.[0-9]+.[0-9]+"
+jobs:
+    publish-release:
+        runs-on: ubuntu-22.04
```

A nitpick, I would use `ubuntu-latest` here, as specific versions can get deprecated over time.

```suggestion
        runs-on: ubuntu-latest
```